namesys/publisher.go

```diff
@@ -44,56 +46,102 @@ func NewRoutingPublisher(route routing.IpfsRouting) Publisher {
 // and publishes it out to the routing system
 func (p *ipnsPublisher) Publish(ctx context.Context, k ci.PrivKey, value path.Path) error {
 	log.Debugf("Publish %s", value)
+	return p.PublishWithEOL(ctx, k, value, time.Now().Add(time.Hour*24))
```

Comment:
maybe add a note for people watching/reading the code that this is a step in between today, and https://github.com/ipfs/specs/tree/master/records